config.json

```diff
@@ -602,7 +602,14 @@
       "core": false,
       "difficulty": 6,
       "slug": "house",
-      "topics": null,
+      "topics": [
+        "arrays",
+        "conditionals",
+        "loops",
+        "strings",
+        "text_formatting",
+        "variables"
```

Comment:
Great suggestions :) The only one I'm not sure about is `variables`. How do you think that relates to this exercise? You'd have to use variables to solve pretty much every exercise so I would only add it as a topic if using variables was in some way particularly important to the exercise. What do you think? :)